eslint.config.js

```diff
@@ -36,6 +61,7 @@ export default antfu(
       'unused-imports/no-unused-vars': 0,
       'unicorn/no-new-array': 0,
       'import-x/consistent-type-specifier-style': 0,
+      'tailwindcss/enforces-shorthand': 0,
```

Comment:
Turned off this rule for all Vue files to prevent shorthand change

The main reason is to align with main shadcn-ui